pkg/virt-controller/watch/vm.go

```diff
@@ -659,6 +651,20 @@ func (c *VMController) handleCPUChangeRequest(vm *virtv1.VirtualMachine, vmi *vi
 		return fmt.Errorf("CPU hotplug is not allowed while VMI is migrating")
 	}
 
+	// If the following is true, MaxSockets was calculated, not manually specified (or the validation webhook would have rejected the change).
+	// Since we're here, we can also assume MaxSockets was not changed in the VM spec since last boot.
+	// Therefore, bumping Sockets to a value higher than MaxSockets is fine, it just requires a reboot.
```

Comment:
The condition is set correctly, but I'm not sure about the comment.
We would still get here if `maxSockets` was specified explicitly and was changed now together with a Scokets change (while new sockets < new MaxSockets) 
Does it make sense?

Reply:
If maxSockets is changed, the general restartRequired will get triggered above and we won't reach this point.

Reply:
+1